app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  validates :name, presence:true
+  validates :address, presence:true
+  validates :city, presence:true
+  validates :state, presence:true
+  validates :zip, presence:true
+  validates :email, presence:true, uniqueness:true
+  validates :password, presence:true
```

Comment:
I think there might supposed to be a `require: true` here... might be worth inquiring as to its purpose...

Reply:
we received and error that 'RequireValidator' was not a thing. we did try though.